Detectors/MUON/Workflow/src/tracks-matcher-workflow.cxx

```diff
@@ -36,6 +46,30 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
 
 WorkflowSpec defineDataProcessing(const ConfigContext& configcontext)
 {
+  WorkflowSpec specs{};
+
+  auto disableRootOutput = configcontext.options().get<bool>("disable-root-output");
+  auto disableRootInput = configcontext.options().get<bool>("disable-root-input");
+  auto useMC = !configcontext.options().get<bool>("disable-mc");
+
   o2::conf::ConfigurableParam::updateFromString(configcontext.options().get<std::string>("configKeyValues"));
-  return WorkflowSpec{o2::muon::getTrackMatcherSpec()};
+
+  if (!disableRootInput) {
+    specs.emplace_back(o2::mch::getTrackReaderSpec(useMC, "mch-track-reader"));
+    specs.emplace_back(o2::mid::getTrackReaderSpec(useMC, "mid-track-reader"));
+  }
+
+  specs.emplace_back(o2::muon::getTrackMatcherSpec("muon-track-matcher"));
+
+  if (!disableRootOutput) {
+    specs.emplace_back(o2::muon::getTrackWriterSpec("muon-track-writer", "muontracks.root"));
+  }
```

Comment:
Would be better to preserve the detector name in the device name. In any case, the writers must be subjected to completion policy ensuring that the TFs are stored in the ordered way (in case of pipe-lining processing).
You should add to the workflow definition a block like:
https://github.com/AliceO2Group/AliceO2/blob/f153b1e14e688b7dcbb47a0a2f9422c9e9f671b6/Detectors/MUON/MCH/Workflow/src/clusters-writer-workflow.cxx#L19-L23
Note that if you preserve the name `muon-track-writer`, you will need either to adapt the line to
```
policies.push_back(CompletionPolicyHelpers::consumeWhenAllOrdered("muon-track-writer.*"));
```
or modify the regex accordingly .

Reply:
Sorry, I missed this recent modification in the writer workflow.

I think it's better to preserve the name with "muon", to do not mix with the MCH track writer. Here it writes the matched MCH-MID tracks (ReconstructionDataFormats/TrackMCHMID.h).

Unfortunately I had to update alidist as well after updating O2, so it will take a while to recompile...